src/libs/Pusher/pusher.ts

```diff
@@ -114,27 +127,22 @@ function getChannel(channelName) {
 
 /**
  * Binds an event callback to a channel + eventName
- * @param {Pusher.Channel} channel
- * @param {String} eventName
- * @param {Function} [eventCallback]
- *
- * @private
  */
-function bindEventToChannel(channel, eventName, eventCallback = () => {}) {
+function bindEventToChannel(channel: Channel | undefined, eventName: string, eventCallback: <T>(data: T) => void = () => {}) {
     if (!eventName) {
         return;
     }
 
-    const chunkedDataEvents = {};
-    const callback = (eventData) => {
+    const chunkedDataEvents: Record<string, ChunkedDataEvents> = {};
+    const callback = (eventData: string | Record<string, unknown>) => {
         if (shouldForceOffline) {
             Log.info('[Pusher] Ignoring a Push event because shouldForceOffline = true');
             return;
         }
 
-        let data;
+        let data: EventData;
         try {
-            data = _.isObject(eventData) ? eventData : JSON.parse(eventData);
+            data = isObject(eventData) ? eventData : JSON.parse(eventData);
```

Comment:
Are we migrating from `lodash` in a scope of TS migration?
```suggestion
            data =  typeof eventData === 'object' ? eventData : JSON.parse(eventData);
```